addbookfunction.php

```diff
@@ -2,12 +2,15 @@
 require 'functions.php';
 $db = startDb('books');
 
-if (!empty($_POST)) {
-    if (!empty($_POST['bookName']) && !empty($_POST['author']) && !empty($_POST['category']) && !empty($_POST['released'])) {
-        $bookInfo = ['name' => $_POST['bookName'], 'author' => $_POST['author'], 'category' => $_POST['category'], 'released' => $_POST['released']];
-        addBook($bookInfo, $db);
+if (array_key_exists('bookName', $_POST)
```

Comment:
check my previous comment. the `&&` goes on the end, not at the start. and the conditions all go on their own lines, not on the line with the `if`